src/fluree/db/datatype.cljc

```diff
@@ -192,12 +195,14 @@
   - JS: a Javascript Date object. NB: If you don't supply a timezone JS will
         assume it's in your current, local timezone according to your device."
   [s]
-  (when-let [matches (re-matches iso8601-datetime-re s)]
-    #?(:clj
-       (if-let [offset         (peek matches)]
+  #?(:clj
```

Comment:
Is top-level-only reader conditionals configurable? My instinct is to push the reader conditionals as far down as we can so as much of the AST is shared between platforms as possible. Since we already have introduced bugs where they diverge, I'd like to minimize the surface area.